radio/src/logs.cpp

```diff
@@ -350,7 +350,11 @@ void logsWrite()
           GET_2POS_STATE(SE),
           GET_3POS_STATE(SF));
 #elif defined(PCBTARANIS) || defined(PCBHORUS)
+#if defined(PCBX7)
```

Comment:
Perhaps X7 could have its own whole section (lilke flamenco) since it has 3 differences.